src/main/resources/templates/allocations.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>Allocations</title>
+    <link rel="stylesheet" th:href="@{/css/styles.css}" />
+</head>
+<body>
+<div th:replace="header :: header"></div>
```

Comment:
I get an error from the allocations.html. I see an issue at line 8, it should be:  
```html
<div th:replace="fragments/header :: header"></div>
```
because the header is in the fragments folder